### DISCUSSION TITLE
Implementing CRUD in the Hospital Management System

### DISCUSSION THREAD

**Discussion by poksyy** [poksyy/hospital-project-backend#43] [Announcements]
We are currently working on the full implementation of CRUD (Create, Read, Update, Delete) using CrudRepository in the backend of the Hospital Management System. CrudRepository simplifies data management by providing built-in methods for basic CRUD operations, eliminating the need for writing custom SQL queries. This implementation will support CRUD operations for key entities such as patients, doctors, and nurses, allowing efficient data handling through standard repository methods like save(), findById(), findAll(), and deleteById().

To ensure everything works smoothly, we are actively testing each operation via Postman.

**Comment by krschan** [comment]
https://www.youtube.com/watch?v=Pw5LH_bW8gw
